src/main/scala/cromwell/engine/backend/BackendCall.scala

```diff
@@ -102,5 +121,11 @@ trait BackendCall {
    * this method will not be called.  If the backend does override `Backend#findResumableExecutions`, the corresponding
    * `BackendCall` should override this method to actually do the resumption work.
    */
-  def resume(jobKey: JobKey): ExecutionResult = ???
+  def resume(jobKey: JobKey)(implicit ec: ExecutionContext): Future[ExecutionHandle] = ???
```

Comment:
While you're touching this line, `Predef.???` was [originally meant](http://www.scala-lang.org/api/2.11.7/index.html#scala.Predef$@???:Nothing) for `// TODO: Implement`, not `throw new NotImplementedError()`.

In this case, based on the comments, I wonder if this should really operate something like

```scala
throw new UnsupportedOperationException(s"resume() called on a non-resumable backend: $this")
```

I admit I am torn, as perhaps this case does justify throwing a we-should-halt-the-VM `java.lang.Error` signal instead.